mmv1/products/apphub/Application.yaml

```
@@ -29,6 +29,7 @@ timeouts:
   update_minutes: 20
   delete_minutes: 20
 autogen_async: true
+autogen_tag: QXBwbGljYXRpb24=
```

I think these should all be autogen_status?

```suggestion
autogen_status: QXBwbGljYXRpb24=
```

---

Hah, was just in 2 files I must have zoned out when typing them